src/main/java/pl/byczazagroda/trackexpensesappbackend/dto/UpdateFinancialTransactionDTO.java

```diff
@@ -0,0 +1,14 @@
+package pl.byczazagroda.trackexpensesappbackend.dto;
+
+import org.springframework.format.annotation.DateTimeFormat;
+
+import javax.validation.constraints.Positive;
+import javax.validation.constraints.Size;
+import java.math.BigDecimal;
+import java.time.Instant;
+
+public record UpdateFinancialTransactionDTO(
+        @Positive BigDecimal amount,
```

Comment:
We have to consider about this validation because we probably want to validate whether number has max 2 digits after comma

Reply:
After a discussion on the sense of the existence of validation as to the amount for a financial transaction, a decision was made that validation is necessary. Please add this.

Reply:
Done